tns-oauth.ts

```diff
@@ -135,6 +159,9 @@ export function loginViaAuthorizationCodeFlow(credentials: TnsOAuthModule.ITnsOA
                     let errSubCode = qsObj['error_subcode'];
                     if (codeStr) {
                         try {
+                            if(credentials.skipNextTokenNav && tokenNavStarted) return true;
```

Comment:
We had a second call to `checkCodeIntercept` that was happening within the `shouldOverrideUrlLoading` method of the webview helper. Dropbox was returning an error because the code had already been used to receive a token by that point. I don't know if that would be an issue for other OAuth providers as well, but I've corrected it here using the `skipNextTokenNav` property, which currently only the Dropbox provider uses.